packages/reports/tests/acceptance/fili-datasource-test.ts

```diff
@@ -147,4 +147,25 @@ module('Acceptance | fili datasource', function (hooks) {
     await click('.navi-report__run-btn');
     assert.dom('.table-row').hasText('2015', 'Results are fetched for Since operator');
   });
+
+  test('Fili Dimension sorting disabled', async function (assert) {
+    assert.expect(4);
+    await visit('/reports/new');
+
+    await clickItem('dimension', 'Date Time');
+    assert
+      .dom('.navi-column-config-base__sort-icon')
+      .doesNotHaveAttribute('disabled', 'A timeDimension column can be sorted');
```

Comment:
will `.isNotDisabled(...)` work here?

Reply:
Nope, tried that unfortunately it checks the type and it has to match one of these

https://github.com/simplabs/qunit-dom/blob/b53ab2614337560d549c094816396ae66163fe7f/lib/assertions/is-disabled.ts#L9-L15